dataikuapi/dss/project.py

```diff
@@ -841,12 +841,11 @@ def list_jupyter_notebooks(self, as_objects=True, active=False):
         :returns: The list of the notebooks - see as_objects for more information
         :rtype: list
         """
-        notebooks = self.client._perform_json("GET", "/projects/%s/jupyter-notebooks/" % self.project_key,
-                                         params={"active": active})
+        notebook_names = self.client._perform_json("GET", "/projects/%s/jupyter-notebooks/" % self.project_key, params={"active": active})
```

Comment:
Your notebook_names is not really a list of string, more `metadata`. An example of printed notebook_name:
[{u'kernelSpec': {u'name': u'python3'}, u'lastModifiedOn': 1616601485243, u'name': u'default', u'language': u'Python', u'creator': u'tian.tian', u'createdOn': 1616601485240, u'tags': [], u'lastModifiedBy': {u'login': u'tian.tian', u'displayName': u'tian.tian'}, u'activeSessions': [{u'notebookName': u'default', u'sessionCreatorDisplayName': u'tian.tian', u'kernelLastActivityTime': 1616598158647, u'sessionId': u'tian.tian__DKU__57eee145-9ee2-41ac-8148-86dc2e175389', u'sessionStartTime': 1616598065590, u'kernelExecutionState': u'idle', u'kernelId': u'3a79aaea-d738-442a-a0eb-db0a8152408f', u'kernelConnections': 1, u'sessionCreator': u'tian.tian', u'projectKey': u'TestPublicAPINotebooks', u'kernelPid': 21350}], u'createdBy': {u'login': u'tian.tian', u'displayName': u'tian.tian'}, u'customFields': {}, u'id': u'default', u'projectKey': u'TestPublicAPINotebooks'}]